pkg/cmd/cli/cmd/deploy.go

```diff
@@ -365,7 +368,7 @@ func (o DeployOptions) cancel(config *deployapi.DeploymentConfig) error {
 				continue
 			}
 
-			deployment.Annotations[deployapi.DeploymentCancelledAnnotation] = deployapi.DeploymentCancelledAnnotationValue
+			deployment.Annotations[deployapi.DeploymentCancelledAnnotation] = unversioned.Now().Format(time.RFC3339)
```

Comment:
hrm. I dunno why I'm changed this (I guess a left over from a rebase). I will move it back.